### ISSUE TITLE
alignment scores outpout

### ISSUE THREAD

**Issue by prubbens** [brianhie/scanorama#160]
When running `integrate()` or `correct()`, a table with values is returned, which I think are the alignment scores between datasets. 

Is this correct? 

If so, I think it would be useful to add this to the output (as it took me some time figuring it out), e.g.: 

      print('Alignment scores between datasets: ')
      print(alignments) 

Of course, if the latter corresponds to the table containing the scores.

**Comment by brianhie** [comment]
That is correct -- the alignment score is based on the percentage of aligned cells. I can certainly consider this as a potential improvement!
